.gitattributes

```diff
@@ -1,2 +1,3 @@
 *.gds filter=lfs diff=lfs merge=lfs -text
 *.spef filter=lfs diff=lfs merge=lfs -text
+TESTBENCH/**/*.v filter=lfs diff=lfs merge=lfs -text
```

Comment:
Currently all the testbenches  are generated from Python scripts. So we no longer need to store them as LFS on github. Verilog files in the TESTBENCH directory is ~4kb. As LFS cost money, I would suggest remove this line.

Reply:
This is just a setting not actual testbench, your previous setting was putting everything on git LFS.
Making it difficult to preview simple verilog files from github.

Reply:
About the v1.0 update I mentioned in the commits, It adds screenshots of top level PnR, and module level PnR files. Nothing changes in op level Netlist or GDS

Reply:
I agree. It is my mistake to put these .v files on git LFS. And I have cleaned up my testbenches. Now all the .v files in the TESTBENCH/*/.v should be ~4kb, a proper size to preview from Github without requiring LFS.

Regarding  your testbenches, they are auto-generated by your task run which are typically large. That is why I suggested to remove them. They are not the one used to run verifications.

Reply:
Which TB paths (*.v) you are referring to.
There are no testbenches added in this pull request

Reply:
- FPGA1212_RESET_HD_SKY_PNR/FPGA1212_RESET_HD_SKY_Verilog/SRC/top_autocheck_top_tb.v
- FPGA1212_RESET_HD_SKY_PNR/FPGA1212_RESET_HD_SKY_Verilog/SRC/top_formal_random_top_tb.v
- FPGA1212_RESET_HD_SKY_PNR/FPGA1212_RESET_HD_SKY_Verilog/SRC/top_top_formal_verification.v

Reply:
Ok, I added `.gitignore` and remove existing `*_tb.v` files

Reply:
Good to go